### DISCUSSION TITLE
第六节课笔记(16班)

### DISCUSSION THREAD

**Discussion by Yang-Changhui** [InternLM/Tutorial#451] [笔记]
16班第六节课笔记提交在下面

**Comment by morcake** [comment]
https://blog.51cto.com/morcake/9355566

**Comment by 2404589803** [comment]
https://2404589803.github.io/My-datawhale-learning/#/shusheng_puyu_learnning53/ch06

**Comment by etheral12138** [comment]
https://y1yc0dwhb0o.feishu.cn/docx/SH60dEJBRoeBrxxdt7bcJmkoncg?from=from_copylink

**Comment by Dhaizei** [comment]
http://t.csdnimg.cn/TqI0f

**Comment by Aitejiu** [comment]
https://www.cnblogs.com/aitejiu/p/17981518

**Comment by zszbyzsz** [comment]
https://juejin.cn/post/7327353593890095155

**Comment by hhhanqian** [comment]
http://xhslink.com/h0Wd6z

**Comment by Durian-1111** [comment]
**Day06-LearningNote**
链接：https://xby48gs45u.feishu.cn/docx/ENDJd2Nj5oKeKCxFmaocGin2nhh?from=from_copylink

**Comment by Star-cre** [comment]
视频公共笔记：https://www.bilibili.com/video/BV1Gg4y1U7uc/ id：戈壁的老王

**Comment by EnableAsync** [comment]
https://enableasync.github.io/internlm/internlm-06/
